composer.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "alleyinteractive/wp-match-blocks",
+    "description": "Match WordPress blocks in the given content.",
+    "type": "library",
+    "license": "GPL-2.0-or-later",
+    "authors": [
+        {
+            "name": "Alley",
+            "email": "info@alley.co"
+        }
+    ],
+    "autoload": {
+	  "classmap": [
+		"src/"
+	  ],
+	  "files": [
+		"src/Alley/WP/match-blocks.php",
+		"src/Alley/WP/Internals/internals.php"
+	  ]
+    },
+    "config": {
+        "optimize-autoloader": true,
+        "classmap-authoritative": true,
+        "allow-plugins": {
+            "alleyinteractive/composer-wordpress-autoloader": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        }
+    },
+    "require": {
+        "php": "^7.4 || ^8.0",
+        "alleyinteractive/laminas-validator-extensions": "dev-feature/supporting-validators"
+    },
+    "require-dev": {
+        "alleyinteractive/alley-coding-standards": "^0.3",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
+        "friendsofphp/php-cs-fixer": "^3.8",
+        "mantle-framework/testkit": "^0.2",
+        "phpunit/phpunit": "^9.5"
```

Comment:
the testkit project will add this for you: https://github.com/alleyinteractive/mantle-framework/blob/9872783fe59c4111628246315bdbdcf8c6fe208a/src/mantle/testkit/composer.json#L15
```suggestion
```

Reply:
Nice catch.